### PR TITLE
XSA-391 CVE-2021-28711 CVE-2021-28712 CVE-2021-28713

### PR DESCRIPTION
**Pull request by xen-project-security-team**
XSA-391 CVE-2021-28711 CVE-2021-28712 CVE-2021-28713

CVE data entry for:
  CVE-2021-28711

(This MR was automatically generated by the Xen Project Security Team's
xensec-internal-tools.  Please let us know if anything could be better.)
  CVE-2021-28712

(This MR was automatically generated by the Xen Project Security Team's
xensec-internal-tools.  Please let us know if anything could be better.)
  CVE-2021-28713

(This MR was automatically generated by the Xen Project Security Team's
xensec-internal-tools.  Please let us know if anything could be better.)

### CODE REVIEW

File: 2021/28xxx/CVE-2021-28711.json [hunk 1]
```diff
@@ -1,18 +1,106 @@
 {
```
Comment (cve-team) [inline 2021/28xxx/CVE-2021-28711.json:1]: {'version_affected': '?', 'version_value': 'consult Xen advisory XSA-391'} is not of type 'array'

File: 2021/28xxx/CVE-2021-28712.json [hunk 2]
```diff
@@ -1,18 +1,106 @@
 {
```
Comment (cve-team) [inline 2021/28xxx/CVE-2021-28712.json:1]: {'version_affected': '?', 'version_value': 'consult Xen advisory XSA-391'} is not of type 'array'

File: 2021/28xxx/CVE-2021-28713.json [hunk 3]
```diff
@@ -1,18 +1,106 @@
 {
```
Comment (cve-team) [inline 2021/28xxx/CVE-2021-28713.json:1]: {'version_affected': '?', 'version_value': 'consult Xen advisory XSA-391'} is not of type 'array'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @xen-project-security-team ,

Can you review your PR and adjust your "affects" object. You can use the [examples provided in V4](https://github.com/CVEProject/cve-schema/blob/master/schema/v4.0/DRAFT-JSON-file-format-v4.md#examples) for guidance. I have also pasted the object below

```
"affects": {
			"vendor": {
				"vendor_data": [
					{
						"vendor_name": " ",
						"product": {
							"product_data": [
								{
									"product_name": "sha-1",
									"version": {
										"version_data": [
											{
												"version_value": " "
											}
										]
									}
								}
							]
						}
					}
				]
			}
		}
```

**Comment by anthnysingleton** [comment]
@xen-project-security-team we are still waiting for the corrections to be made so we can process your request.

**Comment by ijackson** [comment]
@anthnysingleton Oops, thanks, I have fixed the syntax.

**Pull request merged**